libva: build X11 bindings

rwhogg opened the pull request:
autoreconf is only called if wayland is not requested,
but if it is, libtool is needed.

Signed-off-by: Bob W. Hogg <rwhogg@linux.com>

# Contributing to homebrew-xorg:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Linuxbrew/homebrew-xorg/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Linuxbrew/homebrew-xorg/pulls) for the same update/change?
- [ ] Have you included a log of the failed build without your patch using `brew gist-logs <formula>`?
- [ ] Does your submission pass
`brew audit --strict <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

rwhogg committed:
libva: Depends on libtool to build without Wayland

autoreconf is only called if wayland is not requested,
but if it is, libtool is needed.

Signed-off-by: Bob W. Hogg <rwhogg@linux.com>

bw-yuhogg commented:
We need to make sure that the bottle for `libva` includes libva-x11.so and libva-x11.pc, because without them `libvdpau-va-gl` does not build (#289). It might be possible to change that formula to work with Wayland, but I'm not sure how and I don't know much about Wayland.

It seems that `libva` includes the X11 bindings if built in stdenv (on my machine), but not if built in superenv. I'll have to work out what dependencies we need for X11.

Thoughts @maxim-belkin?

rwhogg committed:
libva: Depends on libxext and libxfixes

These also bring in a dependency on libx11, which is also needed.
These are for the X11 bindings, which are needed for libvdpau-va-gl.

Signed-off-by: Bob W. Hogg <rwhogg@linux.com>

bw-yuhogg commented:
I'm not sure if this is still necessary, but the `libtool` dependency is certainly still necessary. I'll close this and reopen one with just the `libtool` change.
